FtcRobotController/src/main/java/com/qualcomm/ftcrobotcontroller/opmodes/SwerveActionDelay.java

```diff
@@ -0,0 +1,60 @@
+package com.qualcomm.ftcrobotcontroller.opmodes;
+
+import com.qualcomm.robotcore.hardware.DcMotor;
+import com.qualcomm.robotcore.util.ElapsedTime;
+
+/**
+ * Created by Steve on 6/26/2015.
+ */
+public class SwerveActionDelay implements SwerveAction {
```

Comment:
Yes, the purpose is to identify Swerve Robotics code. I am fine with changing to something else besides “Swerve”. I don’t care too much about what solution we go with for naming, though SR_ seems a bit cryptic. Not sure I care enough to spend brainpower on finding something better though. :)
Steve

From: Darrell Ross [mailto:notifications@github.com]
Sent: Monday, July 13, 2015 1:37 PM
To: SwerveRobotics/ftc_app
Subject: Re: [ftc_app] MentorSandbox (#3)


In FtcRobotController/src/main/java/com/qualcomm/ftcrobotcontroller/opmodes/SwerveActionDelay.java<https://github.com/SwerveRobotics/ftc_app/pull/3#discussion_r34508536>:

> @@ -0,0 +1,60 @@

> +package com.qualcomm.ftcrobotcontroller.opmodes;

> +

> +import com.qualcomm.robotcore.hardware.DcMotor;

> +import com.qualcomm.robotcore.util.ElapsedTime;

> +

> +/**

> + * Created by Steve on 6/26/2015.

> + */

> +public class SwerveActionDelay implements SwerveAction {

What is the purpose of the "Swerve" prefix? If the intent is to identify Swerve Robotics specific code, I recommend we go with something less ambiguous like "SR_".

"Swerve" is also a description of a drive mode and possibly the description of other items.

—
Reply to this email directly or view it on GitHub<https://github.com/SwerveRobotics/ftc_app/pull/3/files#r34508536>.

This e-mail may contain confidential and/or privileged information. If you are not the intended recipient (or have received this e-mail in error) please notify the sender immediately and destroy this e-mail. Any unauthorized copying, disclosure or distribution of the material in this e-mail is strictly forbidden.